src/asciidoc/docs/_includes/steps-codeeditor.adoc

```diff
@@ -0,0 +1,23 @@
+= Steps for the Code Editor
+
+////
+  Usage:
+    :includespath: ../_includes
+
+    include::{includespath}/steps-codeeditor.adoc[tag=openproject-template2edit]
+    include::{includespath}/steps-codeeditor.adoc[tag=expand-terminal]
+////
+
+
+// tag::all[]
+
+// tag::openproject-template2edit[]
+. In https://withkoji.com/developer/projects[your developer portal page], open the project associated with the template to edit.
```

Comment:
Missing a step. You open the project details page and then need to open the project in the code editor.

Reply:
> 
> 
> As written, there's not much context for these tips on the page. It might be better to add a section about how to open a project the Koji editor earlier in the page. Then, you can add these tips as “from the Koji editor”.

I'd rather do this in a new Jira. Opening a project in the Koji editor would be steps in an include file, which I also need in DC-584. Since I don't know which one would be merged first, I could be stepping on my own toes if I introduce in the same includes in two different PRs. I'm trying to avoid having to resolve merge conflicts or unintended merge results.

Reply:
It's not clear to me how creating a new ticket would resolve these conflicts. The included file would still be needed in both places. I'd rather address the existing ticket and coordinate on priorities / merge schedules.